### ISSUE TITLE
when to update unity sdk to adapt facebook sdk 13 requirementation

### ISSUE THREAD

**Issue by Jason-Zy** [facebook/facebook-sdk-for-unity#613]
my team is using facebook unity sdk 11.0,and we have to adapt facebook sdk 13.0 requirementation as "https://developers.facebook.com/blog/post/2022/02/24/introducing-facebook-platform-sdk-13-related-updates/" says.
when i make some changes in gradle file as follow:
![image](https://user-images.githubusercontent.com/26757867/171367267-c43f344a-a2eb-49c8-9ee8-fbc4a1ea1b8c.png)
everything seems to be ok but the "Facebook.Unity.FB.ShareLink" method always cuases my app crash, my team is surffering this issue and we do need help.
## Checklist

- [ ] I've updated to the latest released version of the SDK
- [ ] I've searched for existing feature requests on [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [ ] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [ ] I've given my issue the title: `Feature Request: [name of my feature request]`

## Goals

What do you want to achieve?

## Expected Results

What do you expect to happen?

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```swift
// INSERT YOUR CODE HERE

var example = "Example code"
```

**Comment by JarWarren** [comment]
We don't have a release date yet, but Unity SDK 13.2 will land this month. In the meantime, you may be able to develop against the `main` branch which already includes some iOS 13.2 updates. Android will follow in the coming days. Otherwise, stay tuned for the official release and I'll update this thread when it's out.

**Comment by zenlita** [comment]
Please find Unity v13.2 here: https://developers.facebook.com/docs/unity/downloads

**Comment by Jason-Zy** [comment]
yes we upgrade to fb unity sdk 13.2 ,but the problem :Facebook.Unity.FB.ShareLink cuase crash is not resolved

**Comment by zenlita** [comment]
We sincerely apologise for that. We have bumped it up to SDK 14.0.0 and in this version, ShareLink will work. Please find Unity v14.0.0 here https://developers.facebook.com/docs/unity/downloads. If there are further issues, please report bugs at https://developers.facebook.com/support/bugs/.
